src/test/groovy/org/gradle/kotlin/test/RelocationTest.groovy

```diff
@@ -62,13 +62,20 @@ class RelocationTest extends Specification {
                 buildscript {
                     repositories {
                         maven {
-                            url "https://dl.bintray.com/kotlin/kotlin-dev"
+                            url "https://dl.bintray.com/kotlin/kotlin-eap"
                         }
                     }
                     dependencies {
                         classpath ('org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinPluginVersion') { force = true }
                     }
                 }
+                root.allprojects {
+                    repositories {
+                        maven {
+                            url "https://dl.bintray.com/kotlin/kotlin-eap"
+                        }
+                    }
+                }
```

Comment:
Please extrac the URL to be used as a variable, so we can update it in one place.

Reply:
Pinging @eskatos